src/app/components/ATIAnalytics/params/index.ts

```diff
@@ -145,10 +225,12 @@ export const buildATIEventTrackingParams = (
     );
 
     return buildParams(data, requestContext, serviceContext);
-  } catch (error) {
+  } catch (error: unknown) {
```

Comment:
```suggestion
  } catch (error: Error) {
```
Any reason we can't type this here? I assume there is but just interested what's going on here